loop/simplepush.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+var request = require('request');
+var dedupeArray = require('./utils').dedupeArray;
+
+/**
+ * Simple client to handle simple push notifications
+ **/
+var SimplePush = function(statsdClient) {
+  this.statsdClient = statsdClient;
+}
+
+SimplePush.prototype = {
+  notify: function(reason, urls, version){
+    if (!Array.isArray(urls)) {
+      urls = [urls];
+    }
+
+    urls = dedupeArray(urls);
+
+    var self = this;
+
+    urls.forEach(function(simplePushUrl) {
+      if (self.statsdClient !== undefined) {
+        self.statsdClient.count("simplepush." + reason, 1);
```

Comment:
I think we can do that directly at the statsd / heka level, but I need to get some more information about that from @kparlante I guess.